lib/auction/auction.go

```diff
@@ -41,10 +41,9 @@ type Bid struct {
 
 // WinningBid contains details about a winning bid.
 type WinningBid struct {
```

Comment:
Oh wait @sanderpick just realized that bidbot doesn't use this struct at all! We could simply move this altogether to the storage broker code to remove dependency - if you think it's okay.

Reply:
ah yeah, good point. I'll move it over to the other PR and update this one